dicesGame/src/main/java/com/example/dices/services/DiceService.java

```diff
@@ -0,0 +1,88 @@
+package com.example.dices.services;
+
+import com.example.dices.models.DiceModel;
+import com.example.dices.repositories.IDiceRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.Random;
+
+@Service
+public class DiceService {
+    private final IDiceRepository diceRepository;
+
+    @Autowired
+    public DiceService(IDiceRepository diceRepository) {
+        this.diceRepository = diceRepository;
+    }
+
+    public static class ServiceConstants {
+        public static final int MINIMUM_DICE_SIZE = 1;
+        public static final int MAXIMUM_DICE_SIZE = 1000;
+
+    }
+
+
+    // show list of created dices
+    public ArrayList<DiceModel> getDices() {
```

Comment:
Please use `List` instead of `ArrayList` as return type.
Please see: https://stackoverflow.com/a/11683073/2420718

Reply:
Changed the return type to List. 
Thanks, now I see that using List is more efficient.